src/components/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,167 @@
+@import '../Media/Media.scss';
+
+.root {
+  position: relative;
+
+  &.block {
+    margin-left: 5px;
+    display: block;
+  }
+
+  &.inline {
+    display: inline-block;
+    vertical-align: middle;
+  }
+}
+
+/* Tooltip */
+.icon {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.popperContainer {
+  opacity: 0;
+  animation: fadein 0.3s ease-in 0.5s forwards;
+
+  .contentBox {
+    @include for-phone-only {
+      display: none;
+    }
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 12px;
+    text-align: center;
+    width: max-content;
+    max-width: 270px;
+    background-color: white;
+    font-size: 12px;
+    padding: 20px var(--Space-24) var(--Space-24) var(--Space-24);
+    box-shadow: 0px 0px 12px 5px #f1f1f1f1;
+    color: black;
```

Comment:
let's use the black we have defined

Reply:
Changed to `var(--GrayDarkHover--translucent)`